lib/experimental/fonts-library/class-wp-fonts-library.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class WP_Fonts_Library {
+
+	const ALLOWED_FONT_MIME_TYPES = array(
+		'otf'   => 'font/otf',
+		'ttf'   => 'font/ttf',
+		'woff'  => 'font/woff',
+		'woff2' => 'font/woff2',
+	);
+
+	/**
+	 * Returns the absolute path to the fonts directory.
+	 *
+	 * @return string Path to the fonts directory.
+	 */
+	static public function get_fonts_directory() {
```

Comment:
Flip the order to `public static`.

Reply:
Done! https://github.com/WordPress/gutenberg/pull/52704/commits/98faaff70aa36c275dc3d1403088097fede35b4b